package/Dockerfile

```diff
@@ -53,7 +53,7 @@ ENV CATTLE_SYSTEM_AGENT_UPGRADE_IMAGE rancher/system-agent:${CATTLE_SYSTEM_AGENT
 ENV CATTLE_SYSTEM_UPGRADE_CONTROLLER_CHART_VERSION 100.0.3+up0.3.3
 
 # System charts minimal version
-ENV CATTLE_FLEET_MIN_VERSION=100.1.0+up0.4.0
+ENV CATTLE_FLEET_MIN_VERSION=100.1.0+up0.5.2-rc.1
```

Comment:
`100.1.0+up0.5.2-rc.1` is only updating the build metadata. It's didn't update the version itself which is 100.1.0 before and after the update. The chart version needs to be incremented.

Isn't the version supposed to be [100.2.2+up0.5.2-rc.1](https://github.com/rancher/charts/tree/dev-v2.6/charts/fleet/100.2.2%2Bup0.5.2-rc.1)?